### PR TITLE
Change FPS references to RWS references

### PR DESCRIPTION
**Pull request by sjledoux**
Now that the "First-Party Sets"  project has been [officially renamed to "Related Website Sets"](https://developer.chrome.com/blog/related-website-sets/), references and filenames should be changed accordingly to refer to Related Websites rather than First-Parties. 

Note: This PR should not be merged until the rest of the pipeline is prepared to handle the name change as well. 

Additional note: This PR will fail the normal runs due to the change in various file names

### CODE REVIEW

File: RWS-Submission_Guidelines.md
Comment (cfredric) [inline RWS-Submission_Guidelines.md:3]:
```suggestion
As Chrome prepares to [ship Related Website Sets](https://groups.google.com/a/chromium.org/g/blink-dev/c/7_6JDIfE1as) to General Availability (targeting a phased roll-out beginning Chrome 113),  we will shift from “testing” to “live” for Related Website Sets submissions. Please note key dates below as they relate to when your submissions will be applied to Stable behavior in Chrome:
```

Not strictly related to the renaming, but the existing grouping looks odd to me.

File: tests/rws_tests.py
Comment (cfredric) [inline tests/rws_tests.py:35]: Not totally sure what `fp` was short for, but should it be updated? Same comment for the other test cases.

File: .github/workflows/rws-submissions-checks.yml
Comment (cfredric) [inline .github/workflows/rws-submissions-checks.yml:35]: This means that the workflow will be broken for this PR only, but should work afterward, right?

File: tests/rws_tests.py
Comment (sjledoux) [reply tests/rws_tests.py]: Fixed the name and tried to make it more descriptive.

File: Getting-Started.md
Comment (sjledoux) [reply Getting-Started.md]: Done.

File: .github/workflows/rws-submissions-checks.yml
Comment (sjledoux) [reply .github/workflows/rws-submissions-checks.yml:35]: Yeah, the current check is failing because it uses the main's version of the submission checks and is failing to find `first-party-sets.json` in this branch. Once this is merged, it shouldn't be a problem though.

Comment (cfredric) [reply .github/workflows/rws-submissions-checks.yml:35]: Right. Since there's another open PR, should we update the workflow so that it can handle either filename? Or are we planning on just asking the PR author to rebase and try again?

Comment (sjledoux) [reply .github/workflows/rws-submissions-checks.yml:35]: It might be best to ask for the rebase. If we update the workflow and accept PRs with `first-party-sets.JSON`, we'd end up with 2 submission lists by accepting the PR and would have to resolve that ourselves, right?

**Review by cfredric — APPROVED** [review]
Approving; go ahead and merge if you're ready.

**Comment by Leonelhdzm99** [comment]
fps a rws

**Comment by Leonelhdzm99** [comment]
#87

**Review by Avalir1996 — COMMENTED** [review]
Pleas i Help you

**Comment by McCollum2024** [comment]
> Now that the "First-Party Sets"  project has been [officially renamed to "Related Website Sets"](https://developer.chrome.com/blog/related-website-sets/), references and filenames should be changed accordingly to refer to Related Websites rather than First-Parties. 
> 
> Note: This PR should not be merged until the rest of the pipeline is prepared to handle the name change as well. 
> 
> Additional note: This PR will fail the normal runs due to the change in various file names
 Vera O McCollum 08/30/2024

**Pull request merged**
